lib/lotus/view/configuration.rb

```diff
@@ -377,9 +379,28 @@ def duplicate
       def load!
         views.each   { |v| v.__send__(:load!) }
         layouts.each { |l| l.__send__(:load!) }
+        load_partials!
         freeze
       end
 
+      # Load partials for each partial template file found under the
+      # given load paths
+      #
+      # @since 0.6.0
+      # @api private
+      def load_partials!
+        Lotus::View::Rendering::PartialTemplatesFinder.new.find_partials(root).each { |key, format, template| add_partial(key, format, template) }
```

Comment:
Could you move this to multiple lines with ```do/end```? i think this would be beter for readability.